clang/include/clang/Driver/Options.td

```diff
@@ -4761,6 +4761,7 @@ def mno_tls_direct_seg_refs : Flag<["-"], "mno-tls-direct-seg-refs">, Group<m_Gr
   HelpText<"Disable direct TLS access through segment registers">,
   MarshallingInfoFlag<CodeGenOpts<"IndirectTlsSegRefs">>;
 def mno_relax_all : Flag<["-"], "mno-relax-all">, Group<m_Group>;
+def mno_sse2avx : Flag<["-"], "mno-sse2avx">, Group<m_Group>;
```

Comment:
I wonder if we should support the negative flag in this PR. GNU assembler supports the CPU-specific optimization flags like `-march=CPU` while LLVM integrated assembler does not.  IIUC, the negative flag is used to disable sse2avx opt while keeping the others.  It seems useless for LLVM.